CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Changelog
+
+The changelog for [Fuel](https://github.com/Fuel/Fuel) is located at the 
```

Comment:
The link is a bit wrong 😃. It is `https://github.com/kittinunf/Fuel/`

Reply:
Oh woops 😅, I'll fix them!